test/users.js

```diff
@@ -0,0 +1,156 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  User = require('../app/models').User;
+
+chai.should();
+
+describe('/users POST', () => {
+  it('should pass sign up, parameters are valid', () => {
+    return chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'Rodrigo',
+        lastName: 'Aparicio',
+        email: 'rodrigo.aparicio@wolox.com.ar',
+        password: '12345678a'
+      })
+      .then(res => {
+        res.should.have.status(200);
```

Comment:
check in the db wether the user was created correctly. This includes making a query to the db and seeing the username and checking wether the password was encrypted correctly.